docs/develop/manifest-element-ordering.md

```diff
@@ -16,37 +16,466 @@ For example, in the `<OfficeApp>` element, the `<Id>`, `<Version>`, `<ProviderNa
 > [!NOTE]
 > The [validator within office-toolbox](../testing/troubleshoot-manifest.md#validate-your-manifest-with-office-toolbox) uses the same error message when an element is out-of-order as it does when an element is under the wrong parent. The error says the child element is not a valid child of the parent element. If you get such an error but the reference documentation for the child element indicates that it *is* valid for the parent, then the problem is likely that the child has been placed in the wrong order.
 
-To find the correct order for the child elements of a given parent element, take the following steps. (This is a simplified process, as XSD files are quite complex. Fully parsing XSD files is out of the scope of this document.)
+The following sections show the manifest elements in the order in which they must appear. There are slight differences depending on whether the `type` attribute of the `<OfficeApp>` element is `TaskPaneApp`, `ContentApp`, or `MailApp`. To keep these sections from becoming too unwieldy, the highly complex `<VersionOverrides>` element is broken out into separate sections.
 
-1. Open the subfolder under [Schemas](https://github.com/OfficeDev/office-js-docs-pr/tree/master/docs/overview/schemas) for the type of add-in that you are creating. 
-2. Open the XSD file where the parent element is defined as a complex type. If you don't know which file has the definition, you may have to do step 3 on multiple files until you find it.
-3. Search for `<xs:complexType name="PARENT_ELEMENT">`, where PARENT_ELEMENT is the name of the parent element.
-4. Inside the definition for the PARENT_ELEMENT, there is (usually) an element called `<xs:sequence>`. The following is the definition for `<SuperTip>` from [TaskPaneAppVersionOverridesV1_0.xsd](https://raw.githubusercontent.com/OfficeDev/office-js-docs-pr/master/docs/overview/schemas/taskpane/TaskPaneAppVersionOverridesV1_0.xsd).
+## Basic task pane add-in element ordering
 
-```xml
```

Comment:
For these xs elements - are these completely removed from add-ins altogether at this point? Just wondering if it is worth mentioning them if they might be present in older add-ins.

Reply:
The xs elements are part of part of basic XSD syntax. They aren't part of a manifest.